R/middleware.R

```diff
@@ -341,9 +350,11 @@ HandlerManager <- R6Class("HandlerManager",
           headers$'Content-Type' <- response$content_type
 
           response <- filter(req, response)
-          return(list(status=response$status,
-            body=response$content,
-            headers=headers))
+          return(list(
+            status = response$status,
+            body = if (head_request) "" else response$content,
```

Comment:
@aronatkins Thanks! I tried explicitly adding `Content-Length` to `headers` with this:

```R
  headers$`Content-Length` <- nchar(response$content, type = "bytes")
```

But then that header comes up twice in the output, once with my value, and once with the computed value.

```
$ curl -I http://localhost:9000/
HTTP/1.1 200 OK
X-UA-Compatible: IE=edge,chrome=1
Content-Type: text/html; charset=UTF-8
Content-Length: 823
Content-Length: 0
```

I'll have to investigate how to avoid this.